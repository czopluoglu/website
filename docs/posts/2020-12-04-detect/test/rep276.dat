11001101001010010011
11001111011101111111
00000000101101100100
00100001001110111110
10000101111110110111
11101101100011010000
01110001100110000110
10000001101011001011
00010111100110111111
00011001011000000011
00100001011010100111
11111001111000100111
10100101010001100110
00101011001100101010
11111111111010111101
01110101001100000111
00001000000010001000
11010011111110111111
00001101101000011111
01000111101010101110
11101111100011111111
01011001000000000001
10010010111001100110
00111000100010010000
11011000010010010011
00100000001101010111
10011001011111100111
10001101000100100011
01011000000111011000
10000000101111110110
01100001000110100100
11111101111111110111
11110101111010000010
01101001101010000010
11100110111000111011
10100001101000010110
11001000110011100011
00001101101111011111
11001001001111111001
00100011001000001011
01001101100110001010
10001011010000001011
10111011111110010111
00100000000000100101
10001001001100111011
11000100011001000011
11000111001100000110
01011011111111010111
01101001101110101111
01010001111010101111
01111101111010100101
11010000000000000000
01111111111110110111
11000101100110010110
11011001110100111110
11001111110110101110
10010111111011110111
00110000101010111011
10100000111111101110
00100010010000100110
10001100111111110110
00111101000110001011
11100000111010110011
00010110111110101111
00010000111010101110
10001000001010111010
10110001111011111110
00110111100101110011
01000101101110001011
01101101000000100011
10010101010110111100
10001101111010010011
11101000011011101110
11010010100010011011
10100000001010000011
11101111111111101110
00000001010110100110
01001110110110111111
00101101100110011010
00010101001111111110
10010011011111111111
10011000011110001001
11010010000111001011
10000100101000101110
01001000110110101101
11111111111101110010
11101111101010111111
10111111011111101101
01100001011010100110
00001100001000100110
00110101110010001011
11010101011000110001
00001101011110001011
00110110100111110111
10010001110110000001
10111111010011001110
10000000011011101110
01101001001111111110
11000000111010010010
10010111111000011011
10100111110010110010
01010110100000100011
10001101000101101111
11111111011111111110
11100010011111101110
00111011011111110011
10001111111111111111
01101100101111111111
01100010001110111111
11010000110110111011
01000110110111100010
00111001101111111010
11111111111110011110
11000000110011101101
01000001011010101001
10110100001110110110
01100101010100111111
00110001110000010111
00010000001010100110
11010000010001000001
10110101111001100100
11011000101100101110
10000101001110000101
10110001110110111111
11101100101001111111
00011001111000111011
11111111111111110010
01000100011010100110
11001000111100111110
01101101110010101001
00101101101011010000
10000111001111101111
11001011011011011101
00100000101010100010
00100001101101010110
11111101010001101000
11001101101111111110
00110100010110001010
00100010000111100111
11111010101110011100
10110001101011111110
11011101001100100110
11111011111110101111
11110000001111101111
00000000001111101110
11100111111111101101
01011001000110000110
10001111000100000010
10000001101000101101
11001111101111100110
11010010111110100111
01101101111111101011
10101110010110110101
00001110011101101011
11011000000000010010
11011101011111101100
10000100001010000100
11100001101100111011
10001101100100000100
11101111111001000110
10111001101010100111
10001011110010111110
00100000011101000010
11111111111011101010
01111111111101111110
01111000011110111011
11010111101011110101
01010001111010110110
10000000010110110011
11101000001000010010
00100001100000100110
11110100111110000111
00001001000101011110
00000000100010100011
00111111111100001111
01110101111011101111
10010000001100001011
01100001100111110011
01100001100000101110
10110000011011101101
10110110110010111110
11110111011110111111
00011111110110011101
11011010101011111111
10101011101111110111
10110111111000000111
11010100010100000011
10001000010010110110
00110100011010001010
11111111111111111110
01100001101101101110
11000000110011011010
11001110101111101101
11000000001111101101
11101111000110000011
00100000111010100010
11101010001100100010
00111101110010110110
00100100111010001010
01001111011111111110
00101100101111100111
00100001010111010101
11011011011110000100
11011001010110000011
10111001010011100010
01000000001010000011
11101000101100111111
11001101000111111101
00000000101010011011
10000101001111110110
10001111111011110001
01101000111111110101
11111101111111100011
00011010001011100010
10000001101010101011
11111111111011110010
00011101011110111111
11111110111111111111
00100110111100100011
01110111100101001111
00011000101000000011
10000100001010111111
10111111111111010111
00011111111110101000
11011111111100100111
10010101101010100101
01010000110011100000
00110000000001110110
11111111111111111110
10000001001000001011
11101010101111101111
11001101111010100010
01001101000011110010
11000010011110010110
10000000011110011101
10001001100011100000
01001000001110100011
11010011111100001100
01100000001110101110
11110100111011111111
01000010000000100110
10010101100000100001
11101011111000101101
11011111111101001111
11111000011110010010
10000011001000010111
01111111111110111111
00001100101010011111
10011011110010001110
01010111110010100100
