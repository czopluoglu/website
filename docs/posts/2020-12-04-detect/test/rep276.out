-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep276.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.60
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7549
                   IDN index value:   0.7000
                           Ratio r:   0.6452

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    1    4    4    1    4    3    1
    5    5    5    5    5    3    5    5    3    1

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    4    7   10   20

   -----------CLUSTER  2-------------

    2

   -----------CLUSTER  3-------------

    3    9   16   19

   -----------CLUSTER  4-------------

    5    6    8

   -----------CLUSTER  5-------------

   11   12   13   14   15   17   18
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d-++++--+-+--+--+--+
   -d+++-++----+-------
   ++d+++-+-+-++-+----+
   +++d++-+---+---+----
   ++++d---+--++++---+-
   +-++-d++--+---+--+--
   -+---+d+-+--+---++--
   -+++-++d++-++--+-+--
   +---+--+d++----++---
   --+---+++d---++--+--
   +----+--+-d++-+-+-+-
   --+++--+--+d+-+-+---
   -++-+-++--++d------+
   +---+----+---d--++++
   --+-++---+++--d++-++
   ---+---++-----+d+++-
   +-----+-+-++-+++d+-+
   -----+++-+---+-++d++
   ----+-----+--+++-+d+
   +-+---------+++-+++d
 
 No cross validation for this DETECT run
